### PR TITLE
Remove AWS `vpn_instance` module

### PR DESCRIPTION
**Pull request by kopiczko**

### CODE REVIEW

File: platforms/aws/giantnetes/variables.tf [hunk 1]
```diff
@@ -301,12 +301,6 @@ variable "external_ipsec_subnet_1" {
   default     = "172.18.0.4/30"
 }
 
-### VPN with custom instance 
-variable "vpn_instance_enabled" {
```
Comment (kopiczko) [inline platforms/aws/giantnetes/variables.tf:305]: Just in case I grepped installations and didn't find it being set anywhere.

**Comment by corest** [comment]
We don't have anyone requesting this now. But if there would be someone with request to have China installation, but not having direct connect - we would still need this.

**Review by corest — APPROVED** [review]

**Comment by kopiczko** [comment]
That's legit. I dropped a card in our inbox to discuss that. But I'm not sure we want to have CN installation without DC.

**Comment by kopiczko** [comment]
We talked and don't want a new CN installation without DC connection set. Otherwise connections are really unreliable.

**Pull request merged**